pkg/controller/migmigration/restore.go

```diff
@@ -222,8 +224,25 @@ func (t *Task) buildRestore(backupName string) (*velero.Restore, error) {
 // Update a Restore as desired for the destination cluster.
 func (t *Task) updateRestore(restore *velero.Restore, backupName string) {
 	restorePVs := true
+
+	namespaceMapping := make(map[string]string)
+	for _, namespace := range t.namespaces() {
+		mapping := strings.Split(namespace, ":")
+		if len(mapping) == 1 {
+			namespaceMapping[namespace] = namespace
+		}
+		if len(mapping) == 2 {
+			if mapping[1] == "" {
+				namespaceMapping[mapping[0]] = mapping[0]
+			} else {
+				namespaceMapping[mapping[0]] = mapping[1]
+			}
+		}
+	}
+
 	restore.Spec = velero.RestoreSpec{
-		BackupName: backupName,
-		RestorePVs: &restorePVs,
+		BackupName:       backupName,
+		RestorePVs:       &restorePVs,
+		NamespaceMapping: namespaceMapping,
```

Comment:
I'm not quite sure whether we should explicitly list an empty mapping like this or whether we should just omit it in the case where we're not changing any namespaces (which we'll know because the mapping will be empty).